installer/templates/phx_single/lib/app_name_web.ex

```diff
@@ -43,8 +47,34 @@ defmodule <%= web_namespace %> do
       import <%= web_namespace %>.Gettext<% end %>
       alias <%= web_namespace %>.Router.Helpers, as: Routes
     end
+  end<%= if live do %>
+
+  def live_view do
+    quote do
+      use Phoenix.LiveView,
+        layout: {<%= web_namespace %>.LayoutView, "live.html"}
+
+      use Phoenix.HTML
+
+      import <%= web_namespace %>.ErrorHelpers<%= if gettext do %>
+      import <%= web_namespace %>.Gettext<% end %>
+      import <%= web_namespace %>.LiveHelpers
+      alias <%= web_namespace %>.Router.Helpers, as: Routes
+    end
   end
 
+  def live_component do
+    quote do
+      use Phoenix.LiveComponent
+      use Phoenix.HTML
+
+      import <%= web_namespace %>.ErrorHelpers<%= if gettext do %>
+      import <%= web_namespace %>.Gettext<% end %>
+      import <%= web_namespace %>.LiveHelpers
+      alias <%= web_namespace %>.Router.Helpers, as: Routes
```

Comment:
There is a good amount of duplication between view, live_component and live_view. We could do this:

```elixir
def live_component do
  quote do
    use Phoenix.LiveComponent
    unquote(view_helpers())
  end
end

defp view_helpers do
  quote do
    use Phoenix.HTML
    import <%= web_namespace %>.ErrorHelpers<%= if gettext do %>
    import <%= web_namespace %>.Gettext<% end %>
    import <%= web_namespace %>.LiveHelpers
    alias <%= web_namespace %>.Router.Helpers, as: Routes
  end
end
```

But then I don't know if we are trying too hard. Thoughts?